prost/src/facade.rs

```diff
@@ -0,0 +1,90 @@
+//! A facade around all the types we need from the `std`, `core`, and `alloc`
+//! crates. This avoids elaborate import wrangling having to happen in every
+//! module.
```

Comment:
I don't like the facade concept. If we keep it, it should work like the `::std::prelude`. But I think it is better to be explicit about your dependencies.